### PR TITLE
doc mroonga_highlight_html: Add translation

### PR DESCRIPTION
**Pull request by HashidaTKS**
There were untranslated sentences.

### CODE REVIEW

**Comment by HashidaTKS** [comment]
@kou, @komainu8 

Would you review this when you have time?

File: doc/locale/ja/LC_MESSAGES/reference.po [hunk 1]
```diff
@@ -2242,12 +2251,14 @@ msgid ""
 "You must specify ``AS query`` to extract keywords from query like the "
 "following::"
 msgstr ""
+"クエリーを使ってキーワードを抽出するには、以下のように ``AS query`` を指定する"
+"必要があります。"
```
Comment (kou) [inline doc/locale/ja/LC_MESSAGES/reference.po:2255]:
```suggestion
"必要があります。::"
```

Comment (HashidaTKS) [reply doc/locale/ja/LC_MESSAGES/reference.po]: Fixed.

**Comment by HashidaTKS** [comment]
@kou 

Thanks, I have addressed your comment.
Please re-review this.

**Pull request merged**